HomeWork/src/main/java/ionio/FileLogger.java

```diff
@@ -0,0 +1,45 @@
+package ionio;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+
+public class FileLogger extends FileLoggerConfiguration {
```

Comment:
FileLogger should not extend a configuration. Think about another class/interface to extend/implement.